### DISCUSSION TITLE
Is there a way to customise the Disclosure component?

### DISCUSSION THREAD

**Discussion by arpitbatra123** [twilio-labs/paste#2932] [Q&A] [answered]
Below is a screenshot of a design that I'm trying to create using Paste. 

![image](https://user-images.githubusercontent.com/21967563/211725916-cfdbb10d-39ff-498f-8f2a-df863e86753c.png)

I need to separate the Tasks list into separate sections and I think the `Disclosure` component is the way to go to do it. However, the Disclosure component is not customizable via props. Is there a way around this? I mainly want to customize the border radius for the heading and padding for the disclosure content. Here's what I've been able to achieve with Paste: 

![image](https://user-images.githubusercontent.com/21967563/211732132-aaefa28a-40ba-4ee0-9048-fcace1dd9d1f.png)

**Comment by nkrantz** [comment, marked as answer]
Hi @arpitbatra123! Our Disclosure component (similar to most of our components) has few modifiable styles. We build our components that way to increase visual cohesion across the Twilio product suite. The package's [Disclosure Content has a set padding](https://paste.twilio.design/components/disclosure/#disclosurecontent).

Because our styles don't always fit every single use case, we also offer primitives for common components, which are accessible and functional basic components without any styling. I think your best bet to achieve your desired disclosure behavior would be to build on top of our [Disclosure Primitive](https://paste.twilio.design/primitives/disclosure-primitive). (Feel free to check out how we've built the [sidebar navigation disclosure components](https://github.com/twilio-labs/paste/tree/main/packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure) for our own documentation site.)

**Reply by arpitbatra123** [reply]
Hey @nkrantz Thanks for answering, I'll check out the Disclosure Primitive!
